xenmgr/Vm/Config.hs

```diff
@@ -694,8 +695,13 @@ pciSpecs :: VmConfig -> Rpc [PciSpec]
 pciSpecs cfg = do
     let devices = vmcfgPciPtDevices cfg
         uuid    = vmcfgUuid cfg
+        pciSysfsResFiles = map (flip (++) "/resource")
```

Comment:
Same with flip() here.  I'm assuming the goal is to prepend "/resource" to a list of sysfs paths?
```haskell
map ("/resource" ++)
```
should work fine.